ro_help/hub/templates/partials/social_sharing.html

```diff
@@ -0,0 +1,13 @@
+{% load social_share %}
+<div id="social-sharing">
+    Share on:
+    <a class="button is-rounded is-small is-facebook" href="{{ facebook_url }}" target="_blank">
+        <span class="fab fa-facebook"></span>&nbsp;
```

Comment:
Is there another option to have some margins?
Using `&nbsp;` is a little hackish.